packages/components/src/components/navbar/navbar.stories.ts

```diff
@@ -11,7 +11,8 @@ export default {
     navbarMenuTarget: "_blank",
     searchBarIsOpen: false,
     searchBarShowCloseButton: true,
-    navbarPositionFixed: false
+    navbarPositionFixed: false,
+    showLogo: true
```

Comment:
do you think we should call it showLogo? cause in reality it refers to the logo + application name.